server/graphql/v2/object/VirtualCard.ts

```diff
@@ -0,0 +1,20 @@
+import { GraphQLInt, GraphQLObjectType, GraphQLString } from 'graphql';
+import { GraphQLDateTime } from 'graphql-iso-date';
+import { GraphQLJSONObject } from 'graphql-type-json';
+
+export const VirtualCard = new GraphQLObjectType({
+  name: 'VirtualCard',
+  description: 'VirtualCard related properties.',
+  fields: () => ({
+    id: { type: GraphQLString },
+    CollectiveId: { type: GraphQLInt },
+    HostCollectiveId: { type: GraphQLInt },
```

Comment:
That doesn't look like things we want to expense, we can eventually replace them by `account` + `host`

Reply:
Cool, I'll implement this refactor in the next PR where I actually use this.